### ISSUE TITLE
Httpx version issues

### ISSUE THREAD

**Issue by RossLote** [roboflow/inference#860]
### Search before asking

- [X] I have searched the Inference [issues](https://github.com/roboflow/inference/issues) and found no similar feature requests.

### Question

I have noticed that you have pinned an old version of httpx. I'm trying to use supabase with my project but cannot use a recent version because of httpx incompatibility

https://github.com/supabase/supabase-py/blob/ad25d2211fe90157b53a98b127309cc3b1ae9241/pyproject.toml#L22

Is there a way you can bump this version?

**Comment by PawelPeczek-Roboflow** [comment]
Yeah, I see the pain, yet on our end we needed to pin `httpx`, due to very strange bug with `anthropics` package once the version was newer - we expect this is the tmp issue and will be fixed soon, then we would be able to un-pin

**Comment by PawelPeczek-Roboflow** [comment]
see the comment under the pin https://github.com/roboflow/inference/blob/6abcb3cf292d42411d3c876e57bc37dad691fd5c/requirements/_requirements.txt#L39

**Comment by PawelPeczek-Roboflow** [comment]
but actually - maybe that's just 0.28.0 version issue... will give it a try

**Comment by PawelPeczek-Roboflow** [comment]
kicked off this PR: https://github.com/roboflow/inference/pull/861
let's see what CI will tell about the change 😆

**Comment by PawelPeczek-Roboflow** [comment]
@RossLote this got released and should work now

**Comment by RossLote** [comment]
Brilliant, thanks.

**Comment by PawelPeczek-Roboflow** [comment]
ok, closing the issue - if we can help with anything else, let us know
